src/pages/bug-bounty.tsx

```diff
@@ -0,0 +1,896 @@
+)
+
+const StyledButton = ({ children, ...props }) => (
+  <ButtonLink me="4" mb="0" {...props}>
+    {children}
+  </ButtonLink>
+)
+
+const ClientIntro = (props: ChildOnlyProp) => (
+  <Text
+    textTransform="uppercase"
+    fontSize="0.875rem"
+    color="text300"
+    fontWeight="600"
+    mt={{ base: "3rem", lg: "0" }}
+    {...props}
+  />
+)
+
+const ClientRow = (props: ChildOnlyProp) => (
+  <Box
+    display="flex"
+    alignItems="center"
+    flexDirection={{ base: "column", lg: "row" }}
+    {...props}
+  />
+)
+
+const Client = (props: ChildOnlyProp) => (
+  <Box m="16" mt="4" mb="12" {...props} />
+)
+
+const HeroCard = (props: ChildOnlyProp) => (
+  <Box
+    display="flex"
+    justifyContent="space-between"
+    flexDirection={{ base: "column", lg: "row" }}
+    ps={{ lg: "0" }}
+    mt={{ base: "-2rem", lg: "0" }}
+    {...props}
+  />
+)
+
+const HeroContainer = (props: ChildOnlyProp) => (
+  <Box
+    flex="1 1 50%"
+    p={{ lg: "8rem 2rem 8rem 2rem", base: "6rem 0 4rem 0" }}
+    {...props}
+  />
+)
+
+const Row = (props: ChildOnlyProp) => (
+  <Box
+    display="flex"
+    alignItems="center"
+    flexWrap={{ base: "nowrap", lg: "wrap" }}
+    {...props}
+  />
+)
+
+const StyledCardContainer = (props: ChildOnlyProp) => (
+  <Box display="flex" flexWrap="wrap" m="2rem -1rem 3rem -1rem" {...props} />
+)
+
+const StyledCard = ({ children, ...props }) => (
+  <Card flex="1 1 464px" m="4" p="6" justifyContent="flex-start" {...props}>
+    {children}
+  </Card>
+)
+
+const StyledGrayContainer = ({ children, ...props }) => (
+  <Box
+    w="full"
+    py="16"
+    px="0"
+    mt="8"
+    mb="12"
+    bg="grayBackground"
+    boxShadow="inset 0px 1px 0px var(--eth-colors-tableItemBoxShadow)"
+    {...props}
+  >
+    {children}
+  </Box>
+)
+
+const Faq = (props: ChildOnlyProp) => (
+  <Box
+    display="flex"
+    mt="16"
+    flexDirection={{ base: "column", lg: "row" }}
+    {...props}
+  />
+)
+const LeftColumn = (props: ChildOnlyProp) => <Box w="full" {...props} />
+
+const RightColumn = (props: ChildOnlyProp) => (
+  <Box
+    w="full"
+    ms={{ base: "0rem", lg: "2rem" }}
+    flexDirection={{ base: "column", lg: "column" }}
+    {...props}
+  />
+)
+
+type BountyHuntersArg = { score?: number }
+
+type Node = {
+  readonly name: string
+  readonly username: string
+  readonly score: number
+}
+
+type Client = {
+  title: string
+  link: string
+  image: ImageProps["src"]
+}
+
+type Spec = {
+  title: ReactNode
+  link: string
+}
+
+const sortBountyHuntersFn = (a: BountyHuntersArg, b: BountyHuntersArg) => {
+  if (!a.score || !b.score) return 0
+  return b.score - a.score
+}
+
+export const getStaticProps: GetStaticProps<{}, {}> = async (context) => {
+  const { locale } = context
+
+  // load i18n required namespaces for the given page
+  const requiredNamespaces = getRequiredNamespacesForPath("bug-bounty")
+
+  return {
+    props: await serverSideTranslations(locale!, requiredNamespaces),
+  }
+}
```

Comment:
We need to pass the `lastDeployDate` prop to display the date correctly.

```suggestion
  const requiredNamespaces = getRequiredNamespacesForPath("bug-bounty")
  const lastDeployDate = getLastDeployDate()

  return {
    props: {
      ...(await serverSideTranslations(locale!, requiredNamespaces)),
      lastDeployDate
    },
  }
}
```